lighthouse-core/lib/sentry.js

```diff
@@ -37,6 +37,9 @@ const sentryDelegate = {
   getContext: noop,
   /** @type {(error: Error, options?: CaptureOptions) => Promise<void>} */
   captureException: async () => {},
+  _shouldSample() {
+    return SAMPLE_RATE >= Math.random();
```

Comment:
heh. check out `sampleRate` on https://docs.sentry.io/clients/node/config/

I think they added it a _while_ ago. I wonder why we never saw it.

Anyway, I think it's fine to use our solution for now. :)